app/Actions/Album/Create.php

```diff
@@ -134,4 +135,16 @@ private function grantFullPermissionsToNewOwner(Album $album)
 
 		$album->access_permissions()->save($access_perm);
 	}
+
+	private function setStatistics(Album $album): void
+	{
+		$album->statistics()->create([
+			'album_id' => $album->id,
+			'photo_id' => null,
```

Comment:
does this have to be provided? the default for `photo_id` is null?

Reply:
good point. :)